lhotse/dataset/audio_tagging.py

```diff
@@ -0,0 +1,137 @@
+from typing import Callable, Dict, List, Union
+
+import torch
+from torch.utils.data.dataloader import DataLoader, default_collate
+
+from lhotse import validate
+from lhotse.cut import CutSet
+from lhotse.dataset.input_strategies import BatchIO, PrecomputedFeatures
+from lhotse.utils import compute_num_frames, ifnone
+from lhotse.workarounds import Hdf5MemoryIssueFix
+
+
+class AudioTaggingDataset(torch.utils.data.Dataset):
+    """
+    The PyTorch Dataset for the audio tagging task.
+
+    .. code-block::
+
+        {
+            'inputs': float tensor with shape determined by :attr:`input_strategy`:
+                      - single-channel:
+                        - features: (B, T, F)
+                        - audio: (B, T)
+                      - multi-channel: currently not supported
+            'supervisions': [
+                {
+                    # For audio event, which can be mapped to a multi-hot tensor
```

Comment:
Don't you prefer to construct the multi-hot tensor inside the dataset instead?

Reply:
Sorry for getting back so late. 

My intention was that users might need the name of the audio event for other tasks, for example, audio captioning.